test-network/configtx/configtx.yaml

```diff
@@ -339,3 +339,19 @@ Profiles:
                 - *Org2
             Capabilities:
                 <<: *ApplicationCapabilities
+    TwoOrgsApplicationGenesis:
+        <<: *ChannelDefaults
+        Orderer:
+            <<: *OrdererDefaults
+            Organizations:
+                - *OrdererOrg
+            Capabilities:
+                <<: *OrdererCapabilities
+        Application:
+            <<: *ApplicationDefaults
+            Organizations:
+                - *Org1
+                - *Org2
+            Capabilities:
+                <<: *ApplicationCapabilities
+                
```

Comment:
Actually, maybe it always complains in github about this. We should remove the empty line 357.